test/test_transforms.py

```diff
@@ -127,43 +98,37 @@ def test_mu_law_companding(self):
         sig_exp = transforms.MuLawExpanding(quantization_channels)(sig_mu)
         self.assertTrue(sig_exp.min() >= -1. and sig_exp.max() <= 1.)
 
-        repr_test = transforms.MuLawEncoding(quantization_channels)
```

Comment:
Was this block meant to be removed?

Reply:
Yes the check was to see if the class has a __repr__. Every class has a __repr__ so it did not  really do anything